docs/install_packages_and_deploy_cluster.md

```diff
@@ -0,0 +1,95 @@
+
+# How to install the packages and deploy local cluster on Linux
+
+To deploy and run a cluster on your Linux development machine, install the runtime and common SDK generated as the build output. 
+
+## Prerequisites
+
+The following operating system versions are supported for development:
+
+* Ubuntu 16.04 (`Xenial Xerus`)
+
+Ensure you have run the build using the -createinstaller option to generate the debian files
+```
+./runbuild.sh -createinstaller
+```
+
+## Installation steps
+
+### Update your APT sources
+To install the SDK and the associated runtime package via the apt-get command-line tool, you must first update your Advanced Packaging Tool (APT) sources.
+
+1. Open a terminal.
+
+2. Add the `dotnet` repo to your sources list.
+
+    ```bash
+    sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet-release/ xenial main" > /etc/apt/sources.list.d/dotnetdev.list'
```

Comment:
Is this because the debian has a package dependency on dotnet?

Reply:
Yes